README.md

```diff
@@ -53,13 +53,20 @@ Optional:
  - kickstart_timezone
  - kickstart_log_host
  - dhcp_kickstart_manage_packages
- - kickstart_packages # if dhcp_kickstart_manage_packages is defined then kickstart_packages must contain %packages
+ - kickstart_packages # if dhcp_kickstart_manage_packages is defined, it must contain at least @Core or @Base
  - kickstart_extra_pre_option
  - kickstart_extra_pre_commands
  - kickstart_extra_post_commands
```

Comment:
dhcp_kickstart_disable_ipv6: True
could be added to the documentation

Reply:
But it is already there (though with "set as False if needed")